### ISSUE TITLE
Including TypoScript files throws tokenization error

### ISSUE THREAD

**Issue by atomicptr** [martin-helmich/typo3-typoscript-lint#59]
Using the include typoscript statement will throw a tokenization error

```
<INCLUDE_TYPOSCRIPT: source="DIR:EXT:extension_key/Configuration/TypoScript/config" />
```

Throws:

```
Completed with 1 issues

CHECKSTYLE REPORT
=> /home/atomicptr/dev/extension_key/Configuration/TypoScript/setup.typoscript.
   1 Tokenization error: Cannot tokenize line "<INCLUDE_TYPOSCRIPT: source="DIR:EXT:extension_key/Configuration/TypoScript/config" />"

SUMMARY
1 issues in total. (1 errors)
```

**Comment by martin-helmich** [comment]
Thanks for this report!

Using `<INCLUDE_TYPOSCRIPT ... />` as a self-closing tag (`/>`) is nothing I've ever seen before (not like this is actual XML, after all). None of the documentation I've found nor the [actual parser's source code](https://github.com/TYPO3/TYPO3.CMS/blob/6f5d20a1c5a7c08915a865306b4f6879ac0be352/typo3/sysext/core/Classes/TypoScript/Parser/TypoScriptParser.php#L829) look like this is actually supported.

In the original parser, a self-closing `INCLUDE_TYPOSCRIPT`-tag will match the first regular expression in line 830, but not the second expression in line 847. So even if this include works in your setup, it will definitely not work as intended as soon as you add a `condition` statement (which, as I think of it, is not supported by my [typo3-typoscript-parser](https://github.com/martin-helmich/typo3-typoscript-parser), either -- that's worth a separate issue).

Since using this syntax (although supported to some extent, but neither documented nor intended) may lead to unexpected surprises, I'm leaning towards continuing to report this as a syntax error. Thoughts?

**Comment by atomicptr** [comment]
Interesting, I've never thought about this statement being wrong since it worked until now. Since as you noted this leads to unexpected behavior, this should probably be reported as a bug in the TypoScript parser, it probably shouldn't work at all.
